src/app/models/proptypes.js

```diff
@@ -1,29 +1,60 @@
-import { arrayOf, shape, string, bool } from 'prop-types';
+import { arrayOf, bool, oneOfType, shape, string } from 'prop-types';
 
-export const imagePropTypes = {
-  model: shape({
-    blocks: arrayOf(
-      shape({
-        locator: string,
-      }),
-    ),
-  }),
+export const textPropTypes = {
```

Comment:
I notice that this is called textPropTypes but it's not used in the Text component and it does define different props to that component. Do you know what is happening here? (This doesn't necessarily apply to this PR).

Reply:
This was named this way before the Text component was created, hence unfortunate naming!
This is used for blocks that use text, such as `altText`, `caption` within Image component and just directly by the SubHeading and Headline components. If you can suggest a better name, I will gladly change it!

Reply:
Naming will be addressed in this PR https://github.com/bbc/simorgh/pull/258

Reply:
I think we should make it clearer, but not in this PR.